src/jekyll/_includes/page-structure/navigation.liquid

```diff
@@ -17,7 +17,7 @@ We don't have a content nav for the root of /web/ so for this we just skip
         {{ contentnavTocItem.title }}
       </a>
       {% endif %}
-      {% if contentnavTocItem.hasSubNav %}
+      {% if contentnavTocItem.hasSubNav == true and contentnavTocItem.isInCurrentSection == true %}
```

Comment:
Do you know what page causes this to be required? Seems like the data structure of the content nav is wrong.